app/Listeners/LogListener.php

```diff
@@ -75,6 +76,15 @@ public function onCheckoutAccepted(CheckoutAccepted $event)
         $logaction->save();
     }
 
+    /**
+     * @throws \Exception
+     */
+    public function onAssetsTransferredInBulk(AssetsTransferredInBulk $event): void
+    {
+        Log::debug('event passed to the listener:');
+        $event->transferable->logCheckout($event->note, $event->transferredTo, $event->transferable->last_checkout, $event->originalValues, true);
```

Comment:
Hit an exception here: Property [last_checkout] does not exist on this collection instance.